src/k8s-engine/lima.ts

```diff
@@ -1840,10 +1840,16 @@ CREDFWD_URL="http://${ hostIPAddr }:${ stateInfo.port }"
 `;
       const credfwdDir = '/etc/rancher/desktop';
       const credfwdFile = `${ credfwdDir }/credfwd`;
+      const configContents = `{
+  "credsStore": "rancher-desktop"
+}
+`;
```

Comment:
```suggestion
      const configContents = `{ "credsStore": "rancher-desktop" }`;
```

Reply:
This code is going to die in branch 2251.